aws/policy/storage-services.yaml

```diff
@@ -32,4 +32,6 @@ Statement:
       - s3:PutBucketNotification
       - s3:GetBucketLogging
       - s3:PutBucketLogging
+      - s3:GetMetricsConfiguration
+      - s3:PutMetricsConfiguration
```

Comment:
Request metrics are billed at CloudWatch pricing so the Put will need to be under a new AllowGlobalUnrestrictedResourceActionsWhichIncurFees Sid.  Otherwise looks good - thanks!